src/openrct2/ride/TrackPaint.h

```diff
@@ -258,6 +258,8 @@ extern const CoordsXY defaultRightQuarterTurn3TilesOffsets[4][3];
 extern const CoordsXYZ defaultRightQuarterTurn3TilesBoundOffsets[4][3];
 extern const CoordsXY defaultRightQuarterTurn3TilesBoundLengths[4][3];
 
+extern const int8_t right_quarter_turn_3_tiles_sprite_map[];
```

Comment:
Can you make this a constexpr instead, using the new kCamelCase notation?

Reply:
Should all of the other constants in this file be changed to the new notations and to use constexpr as well?